### ISSUE TITLE
Accesibility error in mobile top navigation with navigation groups

### ISSUE THREAD

**Issue by ad9242** [mi6/ic-ui-kit#1339]
## Summary of the bug 
The aria-expanded attribute in ic-navigation-group is causing an accessibility failure in storybook & Cypress tests. This is visible when the mobile menu is open

## 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. With the following code in storybook:
```
<IcTopNavigation
  appTitle="ApplicationName"
  status="alpha"
  version="v0.0.7"
>
  <IcNavigationGroup slot="navigation" label="Navigation Group">
    <IcNavigationItem label="One" href="/" selected />
    <IcNavigationItem label="Two" href="/" target="_blank" />
  </IcNavigationGroup>
</IcTopNavigation>
```
2. view on small device size
3. open the top nav menu
4. See the error in the storybook accessibility add in tab:

![image](https://github.com/mi6/ic-ui-kit/assets/116102637/77874dc9-2bb6-4892-8876-5c7ab588798e)

## 📝 Acceptance Criteria  

*Given* there is a top navigation with navigation groups on a mobile device
*When*  the side menu is open
*Then* there should be no accessibility errors

## Additional Info
when this issue is resolved, it should be possible to add the check back into the appropriate top navigation cypress test

**Comment by ASM995** [comment]
Closed as duplicate ticket.
